paddle/pserver/PServerUtil.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
Please refer to [source file naming convention](https://google.github.io/styleguide/cppguide.html#File_Names) and rename this to be `pserver_util.h`.